### ISSUE TITLE
Please add Darkscans for extension

### ISSUE THREAD

**Issue by xZick** [tachiyomiorg/tachiyomi-extensions#13645]
### Source name

Darkscans

### Source link

https://darkscans.com/

### Language

English

### Other details

I found this while trying to access Nitroscans main website but I kept being blocked by Cloudflare like most users online because the extension doesn't update as well. Then I found a site that checks the status of a websites and saw comments that some users are being redirected to this site. So I figured I might as well request this to be added. Thank you.

![2](https://user-images.githubusercontent.com/114753838/193298712-63069c0d-3014-4381-b513-92b956168a45.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

**Comment by w00ogly** [comment]
+1

**Comment by vetleledaal** [comment]
Uses Madara theme
